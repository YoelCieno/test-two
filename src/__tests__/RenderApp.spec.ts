import { expect, it } from 'vitest'
import { render } from 'vue'

it('Render App correctly to snapshot', () => {
  const result = render
  expect(result).toMatchSnapshot()
})