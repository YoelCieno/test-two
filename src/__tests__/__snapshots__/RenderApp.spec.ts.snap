// Vitest Snapshot v1

exports[`Render App correctly to snapshot 1`] = `[Function]`;
